lib-rca/src/main/java/com/facebook/openwifi/librca/models/ClientStats.java

```diff
@@ -0,0 +1,12 @@
+package com.facebook.openwifi.librca.models;
+
+import java.util.List;
+
+/** Statistic Model for each client. Maily handle KPI and metric calculations */
+public class ClientStats {
+    /** Unique ID for each client. */
```

Comment:
Can the javadoc just say `client MAC`?

Reply:
I agree, client MAC is a better description.